### DISCUSSION TITLE
Performance metrics per application or user limit scaling.

### DISCUSSION THREAD

**Discussion by RASS00** [Budibase/budibase#5280] [General]
Well I was wondering recently how much is performance affected when there are n amount of user at the same time in a single application, given that I use multitenancy, would there be a pro, apart from security for creating an app separate especially for this single organization? what is the limit of users? when do you start to see a hit of performance? is there really a difference?

**Comment by melohagan** [comment]
Hey @RASS00,

Budibase is built with scalability in mind. 
For example the internal budibase DB is CouchDB: https://docs.couchdb.org/en/stable/intro/why.html

Performance wise, there are currently around 20,000 tenants on a single cluster in the budibase cloud, and each tenant can have many users.
